componentDeclarations/componentBuilderMethods.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   "columnFamily": {
     "primary": {"name": "primary", "grouping": "column", jsType: "PRIMARY_KEY"},
 
+    "frozen": {"name": "frozen", "grouping": "column", jsType: "frozen"},
```

Comment:
`jsType: "frozen"` should be `jsType: "object"` - see http://docs.datastax.com/en/developer/nodejs-driver/2.1/nodejs-driver/reference/userDefinedTypes.html